src/styles/header.scss

```diff
@@ -0,0 +1,72 @@
+.header {
+  margin: 0 auto;
+  width: 100%;
+  background-color: $bg-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  position: sticky;
+  z-index: 1000;
+
+  &__container {
+    margin: 0 auto;
```

Comment:
The `.header__container` class has a `margin: 0 auto;` property which is not needed. This property is used to center block elements, but in this case, the `.header__container` is a flex container and its children are centered using `justify-content` and `align-items`. So, you can safely remove this line.